libs/shared/src/lib/components/ui/map/map-popup/map-popup.component.ts

```diff
@@ -109,4 +130,32 @@ export class MapPopupComponent
   public zoomToCurrentFeature() {
     this.zoomTo.emit(this.coordinates);
   }
+
+  /**
+   * Create and navigate to the specified url
+   * @param navigateSettings navigation settings
+   */
+  public navigate(navigateSettings: any) {
+    // Closing the popup manually, otherwise, the tooltip isn't destroyed properly
+    this.closePopup.emit();
+    let fullUrl = this.getPageUrl(navigateSettings.pageUrl as string);
+    if (navigateSettings.field) {
+      const field = get(navigateSettings, 'field', '');
+      const value = get(this.feature[this.currValue].properties, field);
+      fullUrl = `${fullUrl}?id=${value}`;
+    }
+    this.router.navigateByUrl(fullUrl);
+  }
+
+  /**
```

Comment:
I only changed this, replaced the fullUrl with `fullUrl = `${fullUrl}?${navigateSettings.field}=${value}`;`